node/src/types/block.rs

```diff
@@ -744,7 +743,7 @@ pub struct Block {
     hash: BlockHash,
     header: BlockHeader,
     body: (), // TODO: implement body of block
-    proofs: Vec<Signature>,
+    proofs: BTreeMap<PublicKey, Signature>,
```

Comment:
This could have been a `ValidatorMap<Option<Signature>>` which would make it a bit more difficult to work with but still correct and more space-efficient.

Reply:
That's true… but if the plan is to move the finality signatures out of the block anyway, it's probably not worth the effort for now.